src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -1849,7 +1851,7 @@ public function guessMimeType(string $path): ?string
         'bdf' => ['application/x-font-bdf'],
         'bdm' => ['application/vnd.syncml.dm+wbxml', 'video/mp2t'],
         'bdmv' => ['video/mp2t'],
-        'bdoc' => ['application/bdoc', 'application/x-bdoc'],
+        'bdoc' => ['application/bdoc', 'application/x-bdoc', 'application/vnd.etsi.asic-e+zip'],
```

Comment:
This change looks wrong to me. the IANA assignment lists `asice` and `sce` as extensions, not `bdoc`

Reply:
> This list is autogenerated from 2 sources:
> 
> * the upstream list of the Apache project
> * customizations maintained in the generation script
> 
> The current contribution would be lost when re-running the script as it edits the generated list directly.

By looking at the update script, it seems to me that old values are never actually removed...

https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Mime/Resources/bin/update_mime_types.php#L63-L64

Reply:
> This change looks wrong to me. the IANA assignment lists `asice` and `sce` as extensions, not `bdoc`

The file container's former extension was `.bdoc` (with the same mime type) with DigiDoc v3, hence I've added it there as well. The new version with DigiDoc v4 is `.asice`.

Here is a bit more context regarding both of these extensions: https://www.id.ee/en/article/what-is-the-difference-between-digitally-signed-documents-with-bdoc-and-asice-extensions/